src/ccpp_constituent_prop_mod.F90

```diff
@@ -171,56 +328,46 @@ end function ccp_properties_get_key
 
    !#######################################################################
 
-   logical function ccp_is_initialized(this, errcode, errmsg)
-      ! Return .true. iff <this> is initialized
-      ! If <this> is *not* initialized and <errcode> and/or <errmsg> is present,
+   logical function ccp_is_instantiated(this, errcode, errmsg)
+      ! Return .true. iff <this> is instantiated
+      ! If <this> is *not* instantiated and <errcode> and/or <errmsg> is present,
       !    fill these fields with an error status
-      ! If <this> *is* initialized and <errcode> and/or <errmsg> is present,
+      ! If <this> *is* instantiated and <errcode> and/or <errmsg> is present,
       !    clear these fields.
 
       ! Dummy arguments
       class(ccpp_constituent_properties_t), intent(in)  :: this
       integer,          optional,           intent(out) :: errcode
       character(len=*), optional,           intent(out) :: errmsg
+      character(len=*), parameter :: subname = 'ccp_is_instantiated'
 
-      ccp_is_initialized = allocated(this%var_std_name)
-      if (ccp_is_initialized) then
-         if (present(errcode)) then
-            errcode = 0
-         end if
-         if (present(errmsg)) then
-            errmsg = ''
-         end if
-      else
-         if (present(errcode)) then
-            errcode = 1
-         end if
-         if (present(errmsg)) then
-            write(errmsg, *) 'ccpp_constituent_properties_t object ',         &
-                 'is not initialized'
-         end if
+      ccp_is_instantiated = allocated(this%var_std_name)
+      call initialize_errvars(errcode, errmsg)
+      if (.not. ccp_is_instantiated) then
+         call append_errvars(1, "ccpp_constituent_properties_t object is not initialized", &
+              subname, errcode=errcode, errmsg=errmsg)
       end if
 
-   end function ccp_is_initialized
+   end function ccp_is_instantiated
```

Comment:
Sorry, this is just me trying to save space with recursive acronyms (hey, I remember the first infinintely-recursive acronym I ran across, GNU, back when it was still pretty new in 1984).
I figured they are harmless because they are all defined within a type with a well defined name so it should be clear what they are for. However, for the record:
`CCP` stands for `C`cpp_`C`onstituent_`P`roperties_t
and 
`CCPT` stands for `C`cpp_`C`onstituent_prop_`P`tr_`T`

These acronyms should not show up outside this module (can they even be called from outside?).